ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -28,7 +28,11 @@
         {
             "cell_type": "code",
             "source": [
-                "#!eval-notebook \"Data/InitSystemorphToMemoryForTesting\""
+                "#!import \"../Constants/CalculationEngine\"",
+                "\nvar constants = $\"#!import \\\"//{projectName}/{environmentName}/Constants/Consts\\\"\";",
```

Comment:
1. Use #!eval-notebook instead of #!import in this line. 
2. Rename constants to evalConstsNotebook

Reply:
@dcolleoni  I tried it now with
![image](https://user-images.githubusercontent.com/120557622/223701574-d526b1c7-0cc9-4510-9126-30283dbe5c7b.png)

and it didn't help: error appeared after the second run. So ProjectionCount=1 is a must.

Reply:
Done.